api/routes/api_user_preferences.py

```diff
@@ -34,6 +33,7 @@ def sport_preferences_endpoint(service: UserService = Provide[Container.user_ser
 
         if request.method == "POST":
             service.add_preferences(dto)
+            delete_recommendation_key(dto.user_id)
```

Comment:
MINOR:

Just for future: It'd be better to do some class, which would handle those caching actions and keep an eye on our TTL. Using plain functions like this is a little bit against OOP, but I'll allow it since we don't have much time left